Source/itemdat.h

```diff
@@ -282,6 +282,7 @@ enum item_cursor_graphic : uint8_t {
 	ICURS_BLITZEN                     = 219,
 	ICURS_DEMON_PLATE_ARMOR           = 225,
 	ICURS_BOVINE                      = 226,
+	ICURS_DEFAULT                     = -1,

```

Comment:
You can't assign -1 to a `uint8_t`

Reply:
Yes you can, it underflows to 255

Reply:
technically he can but it will be 255 xD

Reply:
Well I guess this is pissing off the checks, I'll have to change it :(

Reply:
When I made the comment, it had already failed all the checks lol

Reply:
> When I made the comment, it had already failed ~all~ _most of_ the checks lol

I replied to you before I made it to the bottom of the page :D

Reply:
maybe you could do something like `ICURS_DEFAULT  = 1 << (sizeof(item_cursor_graphic) - 1)`

Reply:
Donezo